src/lib/api/core/base.ts

```diff
@@ -130,7 +134,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
    * and optional restore it when the mediaQueries deactivate
    */
   protected _getDisplayStyle(source?: HTMLElement): string {
-    let element: HTMLElement = source || this._elementRef.nativeElement;
+    let element: HTMLElement = source || this.nativeElement;
```

Comment:
Instead of doing `element = source || this.nativeElement`, you can change the method signature to be `protected _getDisplayStyle(source: = this.nativeElement): string`.

Reply:
I did not know you could initialize with a dynamic property:

```js
protected _getDisplayStyle(source: HTMLElement = this.nativeElement): string {
```

so cool.

Reply:
@crisbeto - note sure if you default value *solves* all the scenarios.
My solution will account for calls like `_getDisplayStyle(null)` where your solution will fail with that.

Reply:
Makes sense, didn't know that scenario had to be handled.